node-rfc/cts

```diff
@@ -0,0 +1,13 @@
+#! /bin/sh
+
+#start the uuid daemon before accessing the library
+uuidd
+
+#This cd needs to be done if you want to use it with Jenkins docker plugin
+cd $SAPBIN
```

Comment:
I doubt the `cd` here will help ...

It would only help if we remember the previous workdir and if we provide the path to the deployable with the old workdir prepended. But IMO this is a dirty approch.

I strongly prefer another approach over this approach where we install all the required node deps with `--global`, also the the parts which are installed transitively (which does not work for the moment). Only in case it turns out that this cannot be achieved we should continue with an approach sketched here.

Reply:
Agreed.
Nevertheless I provide the script with the previous workdir remembered to continue until we have an information about the node deps with ```--global```